### ISSUE TITLE
Parse Slack message history into Fountain (basic)

### ISSUE THREAD

**Issue by louh** [BadIdeaFactory/scriptchat#2]
Slack exports: https://get.slack.help/hc/en-us/articles/201658943-Export-data-and-message-history

At minimum write a very simple converter for messages to Fountain format (we'll do more complicated work later)

**Comment by louh** [comment]
@slifty Can you do a dump of channel history at https://thegpf.slack.com/services/export? And choose a small subset of chats to do a first pass on.

**Comment by slifty** [comment]
Yes -- but I'll be making sure we have permission from folks involved in whatever the channel is!
